### DISCUSSION TITLE
how to add access token in values.yaml

### DISCUSSION THREAD

**Discussion by Lokeshmaxi** [provectus/kafka-ui#2223] [Q&A]
1)how to generate access token for oauth2

2)where to add in in values.yaml"accessTokenAcceptedVersion": 2

**Comment by Haarolean** [comment]
Hi, thanks for reaching out. 

Not quite sure which OAuth flow you're talking about. Can you elaborate, please? We do support implicit and code flows for oauth out of the box.
